templates/aws/aws_cloudformation_template_vm.yml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
The name of the file needs to change.
`vm.cfn.template.yml`

Reply:
noted